tapa/core.py

```diff
@@ -112,7 +112,7 @@
 STATE11 = IntConst("2'b11")
 STATE10 = IntConst("2'b10")
 
-CUSTOM_RTL_FILE_EXTENSIONS = (".v", ".tcl")
+CUSTOM_RTL_FILE_EXTENSIONS = (".v", ".tcl", ".xci")
```

Comment:
also .sv, .svh, .dat, ...
Probably not a good idea to list file extensions, just grab whatever file under the given directory?